# top left
float64 x1
float64 y1
#botom right
float64 x2
float64 y2

float64 depth

